vendor/gems/ruby_spec_helpers/lib/ruby_spec_helpers/capybara_configuration.rb

```diff
@@ -33,11 +33,10 @@
   when :selenium_chrome_headless
     Capybara.register_driver :selenium_chrome_headless do |app|
       args = %w(headless disable-gpu disable-dev-shm-usage no-sandbox window-size=1280,1024)
-      options_key = Selenium::WebDriver::VERSION >= '4' ? :capabilities : :options
 
       Capybara::Selenium::Driver.new app,
         browser: :chrome,
-        options_key => Selenium::WebDriver::Chrome::Options.new(args: args),
+        options: Selenium::WebDriver::Chrome::Options.new(args: args),
```

Comment:
This was to fix this deprecation notice:

```
Selenium [DEPRECATION] [:capabilities] The :capabilities parameter for Selenium::WebDriver::Chrome::Driver is deprecated. Use :options argument with an instance of Selenium::WebDriver::Chrome::Driver instead.
```

But I don't think is actually the fix... Need to sort this one out.

Reply:
It seems like it's working correctly in CI. Are there issues locally?

Reply:
No, no issues locally... the fix just logically doesn't seem right to me but I was reading it at 2am.

Reply:
yea the code you removed seemed to do the opposite of the warning, but if the warning is gone and the code works, great for now.